Requirements.txt

```diff
@@ -0,0 +1,29 @@
+Gebruikersrollen:
+•	Administrateur (read/write) - Hella, Paulus, Robin
+•	Verkoop medewerker (read)
+•	Waarhuis medewerker (read)
+
+Requirements (15)
+
+Functionele eisen
+1.	Als verkoop medewerker wil ik weten welke producten er op voorraad zijn, zodat ik kan inzien welke producten ik kan verkopen
+2.	Als verkoop medewerker wil ik de hoeveelheden per product per dag weten om in te kunnen schatten welke aanbiedingen ik kan uitzetten
+3.	Als administrateur wil ik de prijzen van inkoop per product kunnen bijhouden zodat ik de administratie kloppend kan houden.
+4.	Als administrateur wil ik de prijzen van verkoop bijhouden zodat ik precies de winstamarge kan berekenen en de financiele administratie kloppend kan houden
+5.	Als administrateur wil ik weten hoeveel producten er verkocht zijn zodat de voorraad aangevuld kan worden wanneer dat nodig is
+6.	Als administrateur wil ik personeelsgegevens kunnen aanpassen zodat ik het personeelsbestand kan bijhouden
+7.	Als waarhuis medewerker wil ik de locatie weten van producten zodat ik ze snel kan vinden als ze verkocht zijn
+8.	Als waarhuis medewerker moet ik een melding krijgen van de sales medewerker wanneer hij iets verkocht heeft zodat het optijd bij de klant is
+9.	Als verkoop medewerker wil ik weten welke producten op elkaar aansluiten en samenwerken zodat ik de producten samen kan verkopen
+10.	Als administrateur wil ik de rollen kunnen verdelen in het systeem zodat niet alle medewerkers alle bevoegdheden hebben
+11.	Als administrateur wil ik de hoeveelheden en producten weten die geretourneerd worden zodat ik met het sales team het assortiment kan aanpassen
+12.	Als verkoop medewerker wil ik weten hoeveel producten er nog in aantocht zijn zodat ik mijn klanten hiervan op de hoogte kan stellen als er een product uitverkocht is
+13.	Als administrateur wil ik de personeel gegevens kunnen opvragen zodat ik dit kan terugkoppelen bij vragen of mijn personeel op de hoogte kan brengen van veranderingen.
+
+Niet-Functionele eisen
```

Comment:
Ik snap de a en b niet helemaal. Het moet inderdaad meetbaar zijn. Vaak gebruik je hier ook SMART of INVEST als graadmeter. Maar rationale is me niet duidelijk. Het is ook gewoon prima om het in 1 zin te combineren, of doe anders alle eisen van SMART en niet alleen meetbaar